coding/python_style_guide.rst

```diff
@@ -783,3 +783,36 @@ Iterators and generators SHOULD be used to iterate over large data sets efficien
 -----------------------------------------------------------------------------------
 
 Use iterators, generators (classes that act like iterators) and generator expressions (expressions that act like iterators) to iterate over large data sets efficiently.
+
+`if False:` and `if True:` SHOULD NOT be used
+---------------------------------------------
+
+Code must not be placed inside `if False:` or `if True:` blocks, nor commented out as a block-comment. For debugging code or if you are undecided about which particular implementation to use, such code must be placed inside a "named" `if` statement. Such blocks should have a comment describing why they are disabled. They may have a comment describing the conditions under which said code can be removed (e.g. completion of a ticket, a particular date). For example, for code that will likely be removed in the future, once testing is completed:
+
+.. code-block:: py
+
+    # Delete old_thing() and the below "if" statement once all unittests are finished (DM-123456).
+    use_old_method = False
+    if use_old_method:
+        old_thing()
+    else:
+        new_thing()
+
+It is often beneficial to lift such debugging flags into the method's keyword arguments to allow users to decide which branch to run. For example:
+
+.. code-block:: py
+
+    def foo(x, debug_plots=False):
+        do_thing()
+        if debug_plots:
+            plot_thing()
+
+or, using lsstDebug, which can be controlled as part of a commandline task:
```

Comment:
Maybe put lsstDebug inside double backticks. Eventually this will be a link to the module's documentation page.